packages/babel-jest/src/__tests__/index.ts

```diff
@@ -39,3 +39,39 @@ test(`Returns source string with inline maps when no transformOptions is passed`
   expect(result.map.sources).toEqual(['dummy_path.js']);
   expect(JSON.stringify(result.map.sourcesContent)).toMatch('customMultiply');
 });
+
+test('getCacheKey does not depend on the rootDir', () => {
+  const getCacheKeyArgs = rootDir => {
+    const config = {
+      cwd: rootDir,
+      rootDir,
+    };
+
+    return [
+      '// Some code',
+      `${rootDir}/foo/bar.js`,
+      JSON.stringify(config),
+      {config, instrument: true, rootDir: config.rootDir},
+    ];
+  };
+
+  jest.mock('@babel/core');
```

Comment:
`unmock` at the end of the test? Just to be sure we don't get weird test states if we append more tests in the future